packages/database/src/migrations/20210524212900-FixEntityAggregationsLaosSchools-modifies-data.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+import { updateValues } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+async function getDashboardReportById(db, id) {
+  const { rows: dashboardReports } = await db.runSql(`
+      SELECT * FROM "dashboardReport"
+      WHERE id = '${id}';
+  `);
+  return dashboardReports[0] || null;
+}
+
+// Update data builder configuration for a report
+async function updateBuilderConfigByReportId(db, newConfig, reportId) {
+  return updateValues(db, 'dashboardReport', { dataBuilderConfig: newConfig }, { id: reportId });
+}
+
+const ENTITY_AGGREGATIONS_BY_REPORT_ID = {
```

Comment:
This migration is a lot more generic than it needs to be as I expected to move more than one dashboard report to using entity agg in the db, but as mentioned in the PR description, decided against at this point